content/blog/tikv-3.0ga-release-notes.md

```diff
@@ -0,0 +1,164 @@
+
+## Steady at scale
+
+For the 3.0 release, we've improved TiKV by:
+
+* **Optimizing the Raft heartbeat mechanism.** With the [hibernate region](https://github.com/tikv/rfcs/blob/master/text/2019-03-04-hibernate-raft.md) feature, TiKV now adjusts the heartbeat frequency according to region activity. That means you'll see less CPU time and network traffic from idle regions.
+
+* **Distributing Garbage Collection.** The introduction of a distributed garbage collector improves performance on large scale clusters dramatically, leading to better stability through more consistent performance.
+
+* **Pessimistic Locking.** It's now possible to ask TiKV to enforce transactions using pessimistic locking. This means you can take exclusive ownership over a value for a duration, preventing other requests from modifying it.
+
+* **Expanding our coprocessor.** With lots of new or improved functionalities such as vector operations, batch executor, [RPN](https://en.wikipedia.org/wiki/Reverse_Polish_notation) functions, `work-stealing` thread pool model,  our coprocessor continues to evolve, allowing for more powerful and efficient ways to work with your data.
+
+* **Enhancing operator friendliness.** Human or machine, we've empowered our operators to get more out of TiKV by unifying our log format, adding new features to `tikv-ctl`, adding even more in-depth metrics, and serving HTTP based metrics. This makes TiKV easier to operate, inspect, and enjoy.
+
+* **Refining request types.** While TiKV previously supported commands like `BatchGet`, 3.0 brings a new `BatchCommands` request type. This allows TiKV to handle batches of requests of differing kinds (such as `Get` and `Put`), leading to less data on the wire and better performance. We also added support for raw reversed scanning, and `Insert` semantics on prewrite.
+
+* **Reducing write amplification.** Inspired by the great ideas from [WiscKey](https://www.usenix.org/system/files/conference/fast16/fast16-papers-lu.pdf), we implemented Titan, a key-value plugin that improves write performance for scenarios with value sizes greater than 1KB, and relieves write amplification in certain degrees.
+
+You can see all the changes in detail in the [changelog](https://github.com/tikv/tikv/blob/release-3.0/CHANGELOG.md).
+
+## Improved Performance
+
+Using [`go-ycsb`](https://github.com/pingcap/go-ycsb) we benchmarked TiKV 3.0.0 against TiKV 2.1.14. We benchmarked a cluster of 3 TiKV nodes, 1 PD node, and 1 node running YCSB. We used DigitalOcean `s-8vcpu-32gb` size machines, and you can reproduce the benchmark for yourself using the fully automated terraform script [here](https://github.com/tikv/terraform-tikv-bench).
+
+The settings for YCSB we used were:
+
+* 1KB Value Size
+* 100 Field Length
+* 10 Field Count
+* 3000 Thread Count (per node, so 9000 total)
+
+According to these results below, and the real-world experience of our customers, we're proud to say that TiKV 3.0.0 is the fastest, most usable TiKV yet!
```

Comment:
This section provides no numbers, just setup. Boring. It also starts by mentioning TiKV 2, which is confusing, since this post is about TiKV 3. Better to lead by saying "this release improves performance in these ways under these conditions". Give the big headline numbers straight off the bat. Then go into details if you must, though better to just link to another blog post.

If these numbers are super important than I would also include the big numbers in the very opening of the blog post, as a "this release also improves perf under X scenarios by Y" e.g.

Reply:
We decided to go with a chart. :)